### DISCUSSION TITLE
PNP Search exact wording

### DISCUSSION THREAD

**Discussion by sayuri8** [microsoft-search/pnp-modern-search#2760] [General]
Hello all,

I am not sure if this is only happening on my end or if this is normal. When I search for an item from the list, I have to enter the exact word, otherwise nothing is found.

**Comment by kasperbolarsen** [comment]
So you have to enter "solution" to get a hit, "solut*" returns nothing?

**Reply by sayuri8** [reply]
Exactly

**Reply by kasperbolarsen** [reply]
In pnp-modern-search only or in SharePoint search query tool as well?
